.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -272,10 +276,10 @@ jobs:
             cargo_dir: ~/.cargo
             cc: clang
             cxx: clang++
-          - name: macOS 12 (clang) Qt6
-            os: macos-12
+          - name: macOS 14 (clang) Qt6
+            os: macos-14
             qt_version: 6
-            aqt_version: '6.7.3'
+            aqt_version: '6.8.0'
             aqt_arch: 'clang_64'
             aqt_host: 'mac'
             dyld_framework_path: /Users/runner/work/cxx-qt/Qt/6.7.3/macos/lib
```

Comment:
this needs to change to `dyld_framework_path: /Users/runner/work/cxx-qt/Qt/6.8.0/macos/lib` as the Qt version has changed.

Which solves the cargo tests link failures